.github/workflows/publish-helm-chart.yml

```
@@ -75,6 +75,61 @@ jobs:
             run: |
               helm lint ${{matrix.chartpath}}
 
+          - name: Helm Chart Version Check
+            run: |
+              # old_chart_version diffs commit that triggered workflow against previous 
+              # commit. Pulls out version number of the existing chart or "old chart".
+              # The git diff gives spits out both version numbers, head -2 | tail -1
+              # Grabs the value on the second line. 
+              old_chart_version=$(git diff ${{ github.sha }} \
+               ${{github.event.before}} -- ${{matrix.chartpath}} | 
+               grep 'version:' | awk '/version:/ {print $2}' | head -2 | 
+               tail -1 | tr -d '"')
+
+              # Grabs the chart version number out of the chart that needs
+              # to be checked. 
+              new_chart_version=$(helm show chart ${{matrix.chartpath}} | 
+              grep 'version:' | awk '/version:/ {print $2}')
+
+              # Takes the version numbers and replaces dots with zeroes.
+              old_version=$(echo "$old_chart_version" | awk -F. '{ printf("%d%03d%03d", $1,$2,$3); }')
+              new_version=$(echo "$new_chart_version" | awk -F. '{ printf("%d%03d%03d", $1,$2,$3); }')
+
+              # Null result indicates chart has not changed
+              if [[ -z $old_chart_version ]]; then 
+                echo "Chart version number has not changed." 
+                exit 0
+              fi
```

```suggestion
              # Null result indicates chart has not changed
              if [[ -z $old_chart_version ]]; then 
                echo "Chart version number must be increased." 
                exit 1
              fi
```

If there are changes and the chart version isn't bumped, that's a problem! Also, we should know much earlier if this var is empty, right? Let's move this up to bail as soon as we could possibly know